src/utils.js

```diff
@@ -58,5 +58,22 @@ const fetchComments = ({ article_id }) => {
     });
 };
 
-export { fetchArticles, fetchArticle, fetchComments, updateVote};
+const sendComment = (article_id, user, commentMsg, setComments, setHideError, setPendingSend) => {
+  const commentData = {
+    username: user,
+    body: commentMsg,
+  };
+
+  return newsApi
+    .post(`/articles/${article_id}/comments`, commentData)
+    .then(({ data }) => {
+      setComments((currentComments) => [data.addedComment, ...currentComments]);
+      setHideError(true)
+      setPendingSend(false)
+    }).catch((err)=>{
+        setHideError(false)
+    });
+};
```

Comment:
This sendComment function feels like the opposite of a separation of concerns. Ideally we should have all our API calls in one file and they should/can be imported to where they are needed eg a component or I suppose another utility function should you wish. Maybe an api file and a utils file but not both in one would be best. Just something to think about more than anything 👍